grant-project/grant-monorepo
# Mobile-friendly search filters

Closes #19.

### What This Does

* Removes the `<Affix>` component from filters
    * In general the ant design Affix component is pretty inflexible and janky. I think if we want to keep this behavior, we're better off implementing our own.
* Adds responsive behavior to filters that moves them to a `<Drawer>` component at mobile
* Replaces ant columns with flexbox for better behavior (filters stay a fixed width instead of getting crushed)

### Steps to Test

* Go to /proposals
* Make sure filters look good at desktop size
* Shrink down to mobile, make sure filters disappear
* Click the filters button, make sure filters appear
* Make sure filters work at all sizes

## Screenshots

Closed             |  Open
:-------------------------:|:-------------------------:
<img width="372" alt="screen shot 2018-09-13 at 3 34 02 pm" src="https://user-images.githubusercontent.com/649992/45511833-0e5b1000-b76c-11e8-8294-d95816a576dd.png">  |  <img width="375" alt="screen shot 2018-09-13 at 3 34 19 pm" src="https://user-images.githubusercontent.com/649992/45511841-13b85a80-b76c-11e8-854c-21d2154c8b57.png">
